bin/dfx-network-deploy

```diff
@@ -53,7 +53,7 @@ if [[ "$DFX_NETWORK" == "local" ]]; then
   dfx-canister-set-id --canister_name nns-dapp --canister_id qsgjb-riaaa-aaaaa-aaaga-cai
   dfx-canister-set-id --canister_name internet_identity --canister_id qhbym-qaaaa-aaaaa-aaafq-cai
 
-  dfx-software-internet-identity-install --release "${DFX_II_RELEASE:-pinned}" --flavor "dev" --network local
+  dfx-software-internet-identity-install --release "${DFX_II_RELEASE:-pinned}" --flavor "dev" --network local --mode reinstall
```

Comment:
Might not be needed.  There is a possibility that upgrading will cause stable memory corruption.